x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.test.tsx

```diff
@@ -127,116 +127,148 @@ describe('alerts_list component empty', () => {
 
     wrapper.find('button[data-test-subj="createFirstAlertButton"]').simulate('click');
 
-    // When the AlertAdd component is rendered, it waits for the healthcheck to resolve
-    await new Promise((resolve) => {
-      setTimeout(resolve, 1000);
+    await act(async () => {
+      // When the AlertAdd component is rendered, it waits for the healthcheck to resolve
+      await new Promise((resolve) => {
+        setTimeout(resolve, 1000);
+      });
+
+      await nextTick();
+      wrapper.update();
     });
-    wrapper.update();
+
     expect(wrapper.find('AlertAdd').exists()).toEqual(true);
   });
 });
 
 describe('alerts_list component with items', () => {
   let wrapper: ReactWrapper<any>;
 
+  const mockedAlertsData = [
```

Comment:
I assume this bit was lifted up, to make the resulting `mockResolvedValue()` call a 4-liner.  Which I love! Not a fan of wads of data like this passed literally as parameters, which then makes the call site they are in hard to read/understand. +10!